test cases/windows/8 msvc dll versioning/cp.bat

```diff
@@ -0,0 +1,10 @@
+@ECHO OFF
+@REM FIXME: remove hack when meson handles filenames correctly
+@REM replace '/' with '\', since copy does not like '/'.
+@REM Ideally, meson should pass us the filenames with '\', but
+@REM https://github.com/mesonbuild/meson/pull/737 was reverted.

```

Comment:
Basically, I'm saying I think this PR should remove that line, and simplify the `cp.bat` at the same time.